gp2gp-translator/src/integrationTest/resources/e2e-mapping/output-json/PWTP7_vis-output.json

```diff
@@ -3493,6 +3493,15 @@
             ]
           }
         ],
+        "code": {
+          "coding": [
+            {
+              "system": "http://snomed.info/sct",
+              "code": "196411000000103",
+              "display": "Transfer-degraded record entry"
+            }
+          ]
+        },
```

Comment:
I'm a bit confused as to why we've now got a value for `code` while not having a value previously. 🤔

Reply:
After discussion, have moved the logic into a different section to prevent a degraded coding being added if there is no code in the codeable concept